packages/tooltip/stories.tsx

```diff
@@ -128,12 +125,11 @@ storiesOf('Tooltip Container', module)
 
       return (
         <TooltipContainer
-          tooltipRef={tooltipRef}
           isVisible={boolean('isVisible', false)}
           delayMilliseconds={number('Tooltip delay', 500)}
         >
-          {({ isVisible, getTooltipProps, getTriggerProps }) => {
-            const styles = {
+          {({ isVisible, getTooltipProps, getTriggerProps }: IUseTooltipReturnValue) => {
```

Comment:
`IUseTooltipReturnValue` shouldn't be necessary here if the types are provided correctly in `TooltipContainer`. Could you try this without?

Reply:
The types are provided correctly in `TooltipContainer`, so this is not necessary. Thanks for the suggestion!